### ISSUE TITLE
[BUG]  TypeError Exception during Report generation

### ISSUE THREAD

**Issue by Alb3c** [dfir-iris/iris-web#166]
Describe the bug
Using the default investigation template (https://docs.dfir-iris.org/operations/example_reports/iris_report_template.docx) a TypeError Exception is generated:

2023-01-12 10:54:31 :: ERROR :: app :: log_exception :: Exception on /case/report/generate-investigation/1 [GET]
Traceback (most recent call last):
  File "/opt/venv/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/venv/lib/python3.9/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/venv/lib/python3.9/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/venv/lib/python3.9/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/iriswebapp/app/util.py", line 446, in wrap
    return f(*args, **kwargs)
  File "/iriswebapp/app/blueprints/reports/reports_route.py", line 103, in _gen_report
    fpath = mreport.generate_doc_report(type="Investigation")
  File "/iriswebapp/app/iris_engine/reporter/reporter.py", line 97, in generate_doc_report
    generator = DocxGenerator(image_handler=image_handler)
TypeError: __init__() got an unexpected keyword argument 'image_handler'

To Reproduce

Go to Case > Generate Report 
See Internal Server Error

Expected behavior
No Internal Server Error

**Comment by whikernel** [comment]
@Alb3c Thanks for your report!
May we ask which version of IRIS you're using? If this is version 2.0.0-beta-1, this will be fixed in beta-2.

**Comment by tobraha** [comment]
I can confirm that this is issue remains in version beta-2. It seems that the docx-generator 'image_handler' feature was added to dfir-iris/docx-generator in [#1](https://github.com/dfir-iris/docx-generator/pull/1), but it hasn't been merged into its master branch at this time.

I was able to resolve this by building a wheel package for docx-generator from [develop](https://github.com/dfir-iris/docx-generator/tree/develop) and replacing the extant package in source/dependencies.

I expect that this will be handled before this moves out of beta, but I'd be happy to submit a PR, if you'd like.

Thanks!
-Tommy
